torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -101,10 +111,42 @@ def rref_script_annotation(rref_var):
     return rref_python_annotation(rref_var).to_here()
 
 
+class LocalRRefTest(RpcAgentTestFixture):
+    @dist_init
+    def test_create_local_ivalue_rref(self):
+        if self.rank != 0:
+            return
+
+        # Create a local RRef<MyScriptClass>.
+        rref_script_class = rpc.RRef(MyScriptClass(10))
+        ret = rref_script_class.to_here().get_value()
+        self.assertEqual(ret, 10)
+
+        # Create a local RRef<MyModuleInterface>.
+        rref_script_module = rpc.RRef(MyScriptModule(3))
```

Comment:
nit: in this case, `MyScriptModule`'s constructor arg is no longer `rank`.

Reply:
Let me update it to keep using self.rank.